django_tables2/columns/booleancolumn.py

```diff
@@ -58,6 +59,6 @@ def render(self, value, record, bound_column):
     @classmethod
     def from_field(cls, field):
         if isinstance(field, models.BooleanField):
-            return cls(verbose_name=field.verbose_name, null=False)
+            return cls(verbose_name=title(field.verbose_name), null=False)
         if isinstance(field, models.NullBooleanField):
-            return cls(verbose_name=field.verbose_name, null=True)
+            return cls(verbose_name=title(field.verbose_name), null=True)
```

Comment:
Ah, it indeed makes sense to not move it up, let's keep it as is.

Thanks for trying anyway!